.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+- '7'
```

Comment:
Koa requires 7.6 min, which version travis uses if you define major version only?